### DISCUSSION TITLE
Meshing using multiple threads in windows PC

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/neper#369] [Q&A] [answered]
@rquey I am using Neper version 4.2.1-26 installed on Windows PC using Ubuntu. My simple doubts are (1) why it is always needed to insert the command "export OMP_NUM_THREADS=1" before starting the -M module? (2) Does it mean that meshing cannot be processed using multiple threads in windows based PC? (3) Is there any option to run meshing in multiple threads like the -T module in a windows PC? (4) Whether -part 8 is equivalent to running meshing in 8 threads parallelly? Thanks in advance.

**Comment by rquey** [comment, marked as answer]
(1-3) As far as I know, as a non Windows user, multithreading does not work on Windows, so, yes, you have to run `export OMP_NUM_THREADS=1`(or compile Neper without OpenMP support) so that Neper runs on only 1 core. I have no plan to improve Windows support...

(4) No, `-part 8` partitions the mesh for finite-element simulations. It does now affect the Neper running conditions.

**Reply by sajutabraham** [reply]
Thank you.

**Reply by sajutabraham** [reply]
I have partitioned the hard disc and installed Linux on my PC in dual boot mode and now am able to run meshing with multiple threads (8) :)
